src/main/java/org/broadinstitute/hellbender/engine/SkeletonVariant.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.engine;
```

Comment:
These classes don't belong in the engine package, which is for traversals / transforms and the like, not pure data types. Move to utils/variant.